### DISCUSSION TITLE
😑why cant i use my bot with a fake users name anymore?

### DISCUSSION THREAD

**Discussion by MJEND7** [PrismarineJS/mineflayer#1997] [Q&A] [answered]
Hello, I used mineflayer one year ago and I was not able to join my local server with a fak user name but now I try and it not work. 

here's my code, this is not so much a question of how to fix it but me just wanting to know what happened to this feature.

const mineflayer = require('mineflayer')

const bot = mineflayer.createBot({
	host: '127.0.0.1', 
	port: 25565,
	username: 'John2000',
	version: "1.16.5",
        auth: "mojang"
})

bot.on('chat', (username, message) => {
  if (username === bot.username) return
  bot.chat(message)
})

// Log errors and kick reasons:
bot.on('kicked', console.log)
bot.on('error', console.log)

**Comment by Hazuki-san** [comment]
Remove `auth: "mojang"`

**Reply by MJEND7** [reply]
I will try that thank you.

**Reply by MJEND7** [reply]
No it did not work it just said could not confirme it was a Mojang account

**Reply by amoraschi** [reply]
We need more information, add error and kicked events to see the reason

**Reply by MJEND7** [reply]
The error that is occurring is,

{"translate":"multiplayer.disconnect.unverified_username"} false

**Reply by amoraschi** [reply]
I think that means that the server is in online mode and you're trying to use a cracked account

**Reply by MJEND7** [reply]
Thank you this has resolved the issue

**Comment by u9g** [comment, marked as answer]
The solution was: the server is in online mode and you're trying to use a cracked account
